.env.example

```diff
@@ -2,12 +2,15 @@ CORE_CONTRACT_SRC=${CONTRACT_SRC:-$(pwd)/cairo-contracts/target/dev/starknet_ibc
 COMET_CONTRACT_SRC=${CONTRACT_SRC:-$(pwd)/cairo-contracts/target/dev/starknet_ibc_contracts_CometClient.contract_class.json}
 ERC20_CONTRACT_SRC=${CONTRACT_SRC:-$(pwd)/cairo-contracts/target/dev/starknet_ibc_contracts_ERC20Mintable.contract_class.json}
 ICS20_CONTRACT_SRC=${CONTRACT_SRC:-$(pwd)/cairo-contracts/target/dev/starknet_ibc_contracts_TransferApp.contract_class.json}
-RPC_URL=https://starknet-sepolia.public.blastapi.io/rpc/v0_7
-ACCOUNT_SRC="${HOME}/.starkli-wallets/deployer/account.json"
-KEYSTORE_SRC="${HOME}/.starkli-wallets/deployer/keystore.json"
+
+export STARKNET_RPC="https://starknet-sepolia.public.blastapi.io/rpc/v0_7"
+export STARKNET_ACCOUNT="${HOME}/.starkli-wallets/deployer/account.json"
+export STARKNET_KEYSTORE="${HOME}/.starkli-wallets/deployer/keystore.json"
+export STARKNET_KEYSTORE_PASSWORD=""
+
 COMPILER_VERSION=2.9.1
-KEYSTORE_PASS=<KEYSTORE_PASSWORD>
 
+CORE_CLASS_HASH=""
+COMET_CLASS_HASH=""
 ERC20_CLASS_HASH=""
 ICS20_CLASS_HASH=""
-CONTRACT_ADDRESS=""
```

Comment:
Should we get this `CONTRACT_ADDRESS` back? I see it's getting called within the `invoke.sh`.

Reply:
4718d5e